autoload/ale/fix/registry.vim

```diff
@@ -110,6 +110,11 @@ let s:default_registry = {
 \       'suggested_filetypes': [],
 \       'description': 'Remove all trailing whitespace characters at the end of every line.',
 \   },
+\   'yamlfix': {
+\       'function': 'ale#fixers#yamlfix#Fix',
+\       'suggested_filetypes': ['python'],
```

Comment:
Is this correct? I would expect the suggested filetype to be yaml.

Reply:
You're right @hsanson , fixed in 12eb8d15.

Thanks